src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -800,8 +800,15 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
         }
       })()
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [state.adapters, state.keyring])
+  }, [
```

Comment:
same blocking issue here re: https://github.com/shapeshift/web/pull/7280/files#r1661953092

Reply:
Not valid anymore as discussed we prefer the quickfix regarding the complexity of `WalletProvider`, changed in [5eacb01](https://github.com/shapeshift/web/pull/7280/commits/5eacb015dbea0fa87977f66ebe16bb50b6338894)